Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -129,12 +129,13 @@ class DisplayHymnViewModel: ObservableObject {
             self.isFavorited = isFavorited
         }
     }
+
     func toggleFavorited() {
         isFavorited.map { isFavorited in
             if isFavorited {
-                favoritesStore.deleteFavorite(primaryKey: FavoriteEntity.createPrimaryKey(hymnIdentifier: self.identifier))
+                favoritesStore.deleteTag(primaryKey: FavoriteEntity.createPrimaryKey(hymnIdentifier: self.identifier, tags: "favorited"), tags: "favorited")
             } else {
-                favoritesStore.storeFavorite(FavoriteEntity(hymnIdentifier: self.identifier, songTitle: self.title))
+                favoritesStore.storeFavorite(FavoriteEntity(hymnIdentifier: self.identifier, songTitle: self.title, tags: "favorited"))
```

Comment:
The hardcoded tag should be a little more unique. It's feasible someone might tag their song as "favorited," and it'll suddenly be a Favorite, instead of a song tagged as "favorited." Throw some underscores or something in there to make it less likely someone would pick that as a tag.

Reply:
Addressed in commit 6fd7962 to give it the string "_*_favorited_*_"